src/main/java/net/rpgtoolkit/common/assets/AnimationFrame.java

```diff
@@ -56,16 +56,6 @@ public AnimationFrame(String frameName, long transparentColour, String frameSoun
       } catch (IOException e) {
         e.printStackTrace();
       }
-    } else {
-      // Loads the tileset and fetches the correct tile
-      frameName = frameName.toLowerCase();

```

Comment:
Since tiles are no longer for animation frames, what things are they still good for? Boards?